### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dakota-mewt**
proposed link to Mewt project - Mute/unmute all computer microphones, displays status and hot mic using volume meter

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

- [x] I have read and understood the [Contribution Guidelines](https://github.com/carlosperate/awesome-microbit/blob/master/contributing.md)
- [x] I am making an individual pull request for each suggestion
- [x] I have used the correct spelling and capitalisation for `BBC micro:bit`, `micro:bit`, and `Micro:bit Educational Foundation` (resource title could be excluded if there is a good reason)
- [x] The content linked is in English, or it contains an English translation
- [ ] I have added the new entry to the bottom of its the category
- [x] I have used the following format:
```
- [Resource Title](link) - Resource short Description (2 lines or less in total)
```

### CODE REVIEW

**Comment by dakota-mewt** [comment]
I didn't see the part in the guide about adding entries to the bottom of the categories.  I am seeing it as part of the checklist though.  :(

**Comment by carlosperate** [comment]
Hi @dakota-mewt,
It's listed in the instructions as well: https://github.com/carlosperate/awesome-microbit/blob/master/contributing.md#adding-to-this-list
image

File: README.md [hunk 1]
```diff
@@ -387,6 +387,7 @@ The USB Interface Chip is the microcontroller placed close to the battery connec
 
 All these projects contain steps and resources required for reproduction.
 
+- [Mewt](https://github.com/dakota-mewt/mewt/blob/main/README_Microbit.md) - Mute/unmute all computer microphones, displays status and hot mic using volume meter
```
Comment (carlosperate) [inline README.md:390]: Project looks really cool, thanks for the submission!
Let's move it to the end of the list and we can merge it 👍

**Comment by dakota-mewt** [comment]
Thanks Carlos!  My project contributor asked me to hold off for a short
while so he can get something cleaned up.  I'll cancel the PR until he's
ready.  and when I resubmit, I'll make sure to add it to the end.  Thank
you!

**Comment by carlosperate** [comment]
No worries, just send another PR when it's ready 👍
It'll be a great addition to the list.

**Comment by dakota-mewt** [comment]
yeah, i'm excited to have it out in the world.

it makes use of a bunch of the microbit functionalities and should work
with v1 and v2 (i borrow the v2 microphone LED as an additional status
indicator), and i've even managed to make the PC-side install software prep
both the computer side and also auto-load the mewt hex onto the microbit.
the entire install is literally a <1 minute affair, the toughest part
should be figuring out which side of the USB plug is up.

while solving an actual problem with remote classes.

**Comment by carlosperate** [comment]
Awesome!

**Comment by dakota-mewt** [comment]
Hi Carlos,

I've done the cleanup and re-submitted the PR to add Mewt.

Thank you!

Billy

**Pull request closed without merging**